examples/deployment/docker-compose.yml

```diff
@@ -68,3 +68,10 @@ services:
     depends_on:
       - mysql
 
+# TODO(pavelkalinnikov): Find a way to remove this network declaration. It is
```

Comment:
I wonder if putting the network specific bits in a separate `yml` file and using something like below might work:
```
docker-compose -d examples/deployment/docker-compose.yml -f integration/cloudbuild/network.override.yml up
```
?

Reply:
What is the "-d" flag? Can't find it.

Reply:
Oh wait, that's the detach. But I still don't get how the 2 files a fed to the docker-compose.

Reply:
It seems like you can't combine 2 docker-compose files like that? The "free" parameters are for the service names to start from the the config file, not other docker-compose files.

Reply:
Ah, but you can supply multiple `-f` parameters apparently: https://docs.docker.com/compose/extends/

Reply:
Sorry, typo - that was meant to be `-f` twice >_<

Reply:
Ok, I'll merge this PR for now and try to play with this suggestion.

Reply:
#2350.